libctru/include/3ds/services/cam.h

```diff
@@ -0,0 +1,809 @@
+    EFFECT_MONO     = 0x1,
+    EFFECT_SEPIA    = 0x2,
+    EFFECT_NEGATIVE = 0x3,
+    EFFECT_NEGAFILM = 0x4,
+    EFFECT_SEPIA01  = 0x5,
+} CAMU_Effect;
+
+/**
+ * @brief Camera contrast patterns.
+ */
+typedef enum {
+    CONTRAST_PATTERN_01 = 0x0,
+    CONTRAST_PATTERN_02 = 0x1,
+    CONTRAST_PATTERN_03 = 0x2,
+    CONTRAST_PATTERN_04 = 0x3,
+    CONTRAST_PATTERN_05 = 0x4,
+    CONTRAST_PATTERN_06 = 0x5,
+    CONTRAST_PATTERN_07 = 0x6,
+    CONTRAST_PATTERN_08 = 0x7,
+    CONTRAST_PATTERN_09 = 0x8,
+    CONTRAST_PATTERN_10 = 0x9,
+    CONTRAST_PATTERN_11 = 0xA,
+
+    // Contrast aliases.
+    CONTRAST_LOW    = CONTRAST_PATTERN_05,
+    CONTRAST_NORMAL = CONTRAST_PATTERN_06,
+    CONTRAST_HIGH   = CONTRAST_PATTERN_07,
+} CAMU_Contrast;
+
+/**
+ * @brief Camera lens correction modes.
+ */
+typedef enum {
+    LENS_CORRECTION_OFF   = 0x0,
+    LENS_CORRECTION_ON_70 = 0x1,
+    LENS_CORRECTION_ON_90 = 0x2,
+
+    // Lens correction aliases.
+    LENS_CORRECTION_DARK   = LENS_CORRECTION_OFF,
+    LENS_CORRECTION_NORMAL = LENS_CORRECTION_ON_70,
+    LENS_CORRECTION_BRIGHT = LENS_CORRECTION_ON_90,
+} CAMU_LensCorrection;
+
+/**
+ * @brief Camera image output formats.
+ */
+typedef enum {
+    OUTPUT_YUV_422 = 0x0,
+    OUTPUT_RGB_565 = 0x1,
+} CAMU_OutputFormat;
+
+/**
+ * @brief Camera shutter sounds.
+ */
+typedef enum {
+    SHUTTER_SOUND_TYPE_NORMAL    = 0x0,
+    SHUTTER_SOUND_TYPE_MOVIE     = 0x1,
+    SHUTTER_SOUND_TYPE_MOVIE_END = 0x2,
+} CAMU_ShutterSoundType;
+
+/**
+ * @brief Image quality calibration data.
+ *
+ * TODO: Field documentation.
+ */
+#pragma pack(1)
+typedef struct {
+    s16 aeBaseTarget;
+    s16 kRL;
+    s16 kGL;
+    s16 kBL;
+    s16 ccmPosition;
+    u16 awbCcmL9Right;
+    u16 awbCcmL9Left;
+    u16 awbCcmL10Right;
+    u16 awbCcmL10Left;
+    u16 awbX0Right;
+    u16 awbX0Left;
+} CAMU_ImageQualityCalibrationData;
+
+/**
+ * @brief Stereo camera calibration data.
+ */
+#pragma pack(1)
+typedef struct {
+    /**
+     * @brief Calibration data flags.
+     */
+    #pragma pack(1)
+    struct Flags {
+        /**
+         * @brief Whether the X and Y rotation data is valid.
+         */
+        bool isValidRotationXY;
+        /**
+         * @brief Reserved flags for future use.
+         */
+        bool reservedFlag[3];
+    } flags;
+    /**
+     * @brief Scale to match the left camera image with the right.
+     */
+    float scale;
+    /**
+     * @brief Z axis rotation to match the left camera image with the right.
+     */
+    float rotationZ;
+    /**
+     * @brief X axis translation to match the left camera image with the right.
+     */
+    float translationX;
+    /**
+     * @brief Y axis translation to match the left camera image with the right.
+     */
+    float translationY;
+    /**
+     * @brief X axis rotation to match the left camera image with the right.
+     */
+    float rotationX;
+    /**
+     * @brief Y axis rotation to match the left camera image with the right.
+     */
```

Comment:
If you only give a brief description use "type member;///< description" syntax. It will be way easier to browse it